torq.q

```diff
@@ -583,11 +590,18 @@ if[@[value;`.ps.loaded;0b]; .ps.initialise[]]
 // initialise connections
 if[@[value;`.servers.STARTUP;0b]; .servers.startup[]]
 
+// function to execute functions in .proc.initlist
+.proc.init:{if[count .proc.initlist;{[a]@[value;a;{[x;a].lg.e[`init;"unable to run initialisation function for ",a," - ",x," error"]}[;a]]} each .proc.initlist;.proc.initlist:()]};
```

Comment:
Split this up over multiple lines, it's hard to read as is.

Reply:
When I split it up should I do double space indentation or do I keep it consistent with the rest of the script?

Reply:
Looks like indentation is not consistent across this file, possibly because it's a mix of tabs and spaces. Try copy that style if possible.